test/test-on-remote-ci.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Run end 2 end tests without polluting this repository
+TRAVIS_PULL_REQUEST=4
+TRAVIS_BRANCH="travis"
+TRAVIS_E2E_TOKEN="jZ_gXkKfYtRmbExdfzqE7A"
```

Comment:
Maybe we move these to the travis env VARS

Reply:
These are all hardcoded for testing purposes. Ofc I'd change that later on :)